percy/sitemap.js

```diff
@@ -0,0 +1,37 @@
+const Axios = require("axios")
+const PercyScript = require("@percy/script");
+const parseUrl = require("url-parse")
+const xml = require("xml2json")
+
+async function loadSitemap(url) {
+  const response = await Axios.get(url, {
+    responseType: "text",
+  })
+
+  return xml.toJson(response.data, {
+    object: true,
+  }).urlset.url
+}
+
+
+(async() => {
+
+  console.log('[INFO] Fetching sitemap');
+  const pageReferences = await loadSitemap(process.env.SITEMAP_URL)
+
+  console.log('[INFO] Found %d page references', pageReferences.length);
+  for (const i in pageReferences) {
+
```

Comment:
Can you use a variable to exit the loop after X iteration (like for now to 100 ?) we can use an env variable for that.

Reply:
Yup, pass MAX_PAGES env var to limit number of pages snapshotted.

Reply:
@ekinoben can you default to 100? Thanks